shared/features/contributions/contribution-sidepanel-title/contribution-sidepanel-title.tsx

```diff
@@ -0,0 +1,21 @@
+import { Typo } from "@/design-system/atoms/typo";
+import { ContributionBadge } from "@/design-system/molecules/contribution-badge";
+
+import { ContributionSidepanelTitleProps } from "./contribution-sidepanel-title.types";
+
+export function ContributionSidepanelTitle({ children, badge }: ContributionSidepanelTitleProps) {
+  return (
+    <div className={"flex w-full flex-row items-center justify-start gap-lg overflow-hidden"}>
+      {!!badge && <ContributionBadge {...badge} />}
```

Comment:
Can't use the side panel header `startContent` for this ?

Reply:
will try

Reply:
Nope the start content will centre the title as it's done with the back button